singer-io/tap-github
# TDL-19527 added backoff for 5xx error and added custom error message when only organization is passed in config

# Description of change
- Added backoff for 5xx errors
- Added custom error messages when only organization is passed in config.

# Manual QA steps
 - Verify that the tap backoffs for 5 times for 5xx error.
 - Verify the tap throws proper message when repo name not found in config
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
